src/components/Card.js

```diff
@@ -1,8 +1,12 @@
+import PopupDeleteConfirm from "./PopupDeleteConfirm";
+import { api } from "./Api";
```

Comment:
Please, delete these imports

From the 8th sprint it's not allowed to import anything to the classes. Only class `Popup` can be imported for extending. **`Loose coupling`** should be used: it means that you need to pass callbacks to the constructor rather than using outside elements or functions.